src/index.js

```diff
@@ -44,9 +44,20 @@ function isSupported(feature, browsers) {
     }
   }
 
-  return browserslist(browsers, {ignoreUnknownVersions: true})
-    .map((browser) => browser.split(" "))
-    .every((browser) => data.stats[browser[0]] && data.stats[browser[0]][browser[1]] === "y")
+  const browserList = browserslist(browsers, {ignoreUnknownVersions: true})
+
+  if (browserList && browserList.length > 0) {
+    return browserList.map((browser) => {
+      return browser.split(" ")
+    })
+    .every((browser) => {
+      return data.stats[browser[0]] &&
```

Comment:
Support case insensitive search for "Y":

```
return data.stats[browser[0]] && 
   data.stats[browser[0]][browser[1]] &&
   String.prototype.toLowerCase(data.stats[browser[0]][browser[1]][0]) === "y"
```

Reply:
Also, you should incorporate the fix from https://github.com/Nyalab/caniuse-api/pull/61/files here, so:

```
return data &&
   data.stats[browser[0]] && 
   data.stats[browser[0]][browser[1]] &&
   String.prototype.toLowerCase(data.stats[browser[0]][browser[1]][0]) === "y"
```